PYME/Acquire/ui/AnalysisSettingsUI.py

```diff
@@ -28,22 +28,27 @@ def __init__(self, parent, analysisSettings, mdhChangedSignal=None):
         self.fitFactories = PYME.localization.FitFactories.resFitFactories
 
         hsizer = wx.BoxSizer(wx.HORIZONTAL)
+        self.cbLocalizeInRealTime = wx.CheckBox(self, -1, 'Localize in real time')
+        self.cbLocalizeInRealTime.SetValue(False)
+        self.cbLocalizeInRealTime.Bind(wx.EVT_CHECKBOX, lambda e: self.analysisSettings.SetPropagate(e.IsChecked()))
```

Comment:
I'm not at all sold on the renaming of this checkbox without changing function. As it is, this decides whether to write analysis settings to the metadata (as it says in the label), nothing more, nothing less.

Later on in `LaunchAnalysis` we decide  based on whether settings are being written whether we can automatically analyse. This is a bit arse about face, and is a consequence of the autolaunching being cobbled in quickly to get something working. There really should be a dedicated option for `automatically_launch_analysis`. As we are going to the effort of revisiting this I think we should do it properly.

`automatically_launch_analysis==True` should imply `propagate=True`, but it's a little unclear to me if the converse should also apply. One of the original use cases was to reduce the number of settings you needed to change in dh5view for each file - for example if you were doing a bunch of ratiometric 3D imaging in a session, you could set the fit factory and PSF settings in PYMEAcquire, and would only need to double check the threshold in dh5view before hitting go.

PS - I think the checkbox label should be `Auto-launch localization`, `Automatically start localising` or something similar as whether it is real time or not is a function of your data rate, fit type, and whether you have a cluster or not.

Reply:
The naming is in line with the static text for that section, which is indeed a bit ambitious and should probably also change to automatic.
![Screen Shot 2020-05-24 at 6 22 17 PM](https://user-images.githubusercontent.com/31105780/82766248-8146d600-9deb-11ea-92e9-e3dc5d63fa21.png)

LaunchAnalysis is definitely a bit arse about face. Explaining to a user the nuance of propagating metadata, and what that means for each of the spool types is something I'd like to get around at some point. re: doing it properly, I'm spreading things out over multiple PRs. Ultimately I want the auto analysis to work as a list of chained rules, which we've discussed, but I don't think you want all of in one PR.